Disable code sections which set CFLAGS="${CFLAGS} -Werror=foo"
by setting the variable "found" to "yes" instead of "no"
Stefan Kuhn <wuodan@hispeed.ch> 20120727
--- configure	2012-03-08 05:29:58.000000000 +0100
+++ configure	2012-07-27 22:18:18.104000011 +0200
@@ -13695,7 +13695,7 @@
 	CFLAGS="$xorg_testset_save_CFLAGS"
 fi
 
-found="no"
+found="yes"
 
 	if test $found = "no" ; then
 		if test "x$xorg_testset_unknown_warning_option" = "xyes" ; then
@@ -13860,7 +13860,7 @@
 	CFLAGS="$xorg_testset_save_CFLAGS"
 fi
 
-found="no"
+found="yes"
 
 	if test $found = "no" ; then
 		if test "x$xorg_testset_unknown_warning_option" = "xyes" ; then
@@ -13976,7 +13976,7 @@
 	CFLAGS="$xorg_testset_save_CFLAGS"
 fi
 
-found="no"
+found="yes"
 
 	if test $found = "no" ; then
 		if test "x$xorg_testset_unknown_warning_option" = "xyes" ; then
@@ -14092,7 +14092,7 @@
 	CFLAGS="$xorg_testset_save_CFLAGS"
 fi
 
-found="no"
+found="yes"
 
 	if test $found = "no" ; then
 		if test "x$xorg_testset_unknown_warning_option" = "xyes" ; then
@@ -14208,7 +14208,7 @@
 	CFLAGS="$xorg_testset_save_CFLAGS"
 fi
 
-found="no"
+found="yes"
 
 	if test $found = "no" ; then
 		if test "x$xorg_testset_unknown_warning_option" = "xyes" ; then
@@ -14324,7 +14324,7 @@
 	CFLAGS="$xorg_testset_save_CFLAGS"
 fi
 
-found="no"
+found="yes"
 
 	if test $found = "no" ; then
 		if test "x$xorg_testset_unknown_warning_option" = "xyes" ; then
@@ -14440,7 +14440,7 @@
 	CFLAGS="$xorg_testset_save_CFLAGS"
 fi
 
-found="no"
+found="yes"
 
 	if test $found = "no" ; then
 		if test "x$xorg_testset_unknown_warning_option" = "xyes" ; then
@@ -14605,7 +14605,7 @@
 	CFLAGS="$xorg_testset_save_CFLAGS"
 fi
 
-found="no"
+found="yes"
 
 	if test $found = "no" ; then
 		if test "x$xorg_testset_unknown_warning_option" = "xyes" ; then
@@ -14721,7 +14721,7 @@
 	CFLAGS="$xorg_testset_save_CFLAGS"
 fi
 
-found="no"
+found="yes"
 
 	if test $found = "no" ; then
 		if test "x$xorg_testset_unknown_warning_option" = "xyes" ; then
@@ -14837,7 +14837,7 @@
 	CFLAGS="$xorg_testset_save_CFLAGS"
 fi
 
-found="no"
+found="yes"
 
 	if test $found = "no" ; then
 		if test "x$xorg_testset_unknown_warning_option" = "xyes" ; then
@@ -14953,7 +14953,7 @@
 	CFLAGS="$xorg_testset_save_CFLAGS"
 fi
 
-found="no"
+found="yes"
 
 	if test $found = "no" ; then
 		if test "x$xorg_testset_unknown_warning_option" = "xyes" ; then
@@ -15069,7 +15069,7 @@
 	CFLAGS="$xorg_testset_save_CFLAGS"
 fi
 
-found="no"
+found="yes"
 
 	if test $found = "no" ; then
 		if test "x$xorg_testset_unknown_warning_option" = "xyes" ; then
@@ -15234,7 +15234,7 @@
 	CFLAGS="$xorg_testset_save_CFLAGS"
 fi
 
-found="no"
+found="yes"
 
 	if test $found = "no" ; then
 		if test "x$xorg_testset_unknown_warning_option" = "xyes" ; then
@@ -17010,7 +17010,7 @@
 	CFLAGS="$xorg_testset_save_CFLAGS"
 fi
 
-found="no"
+found="yes"
 
 	if test $found = "no" ; then
 		if test "x$xorg_testset_unknown_warning_option" = "xyes" ; then
@@ -17178,7 +17178,7 @@
 	CFLAGS="$xorg_testset_save_CFLAGS"
 fi
 
-found="no"
+found="yes"
 
 	if test $found = "no" ; then
 		if test "x$xorg_testset_unknown_warning_option" = "xyes" ; then
